pingpong/ai.py

```
@@ -3632,13 +3636,25 @@ async def run_response(
 
             async def _tts_cleanup() -> None:
-                nonlocal _tts_client, _tts_audio_task
+                nonlocal _tts_client, _tts_audio_task, _tts_connect_task
+                if _tts_connect_task and not _tts_connect_task.done():
+                    _tts_connect_task.cancel()
+                    try:
+                        await _tts_connect_task
+                    except asyncio.CancelledError:
+                        # Expected after explicitly canceling the connect task.
+                        pass
+                    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)